css/style.css

```diff
@@ -0,0 +1,117 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800&family=Poppins:wght@200;300;400;500;600;700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+  letter-spacing: 0.1em;
+  display: inline-block;
+  font-weight: 100;
+  padding: 20px 5px;
+  width: 100%;
+  text-align: center;
+  transition: all 0.5s;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  font-family: "Inter", sans-serif;
+}
+
+.nav-flex {
+  text-align: center;
+  margin: 0;
+  padding-right: 2%;
+  padding-left: 2%;
+  max-width: 375px;
+  border: 10px solid black;
```

Comment:
- [ ] Kindly remove this border outline so your design looks like the original 👍🏿